apps/web/components/ui/AvatarGroup.tsx

```diff
@@ -18,23 +18,50 @@ export type AvatarGroupProps = {
 
 export const AvatarGroup = function AvatarGroup(props: AvatarGroupProps) {
   return (
-    <ul className={classNames(props.className)}>
-      {props.items.slice(0, props.truncateAfter).map((item, idx) => {
-        if (item.image != null) {
-          return (
-            <li key={idx} className="-mr-2 inline-block">
-              <Avatar
-                className={props.border}
-                imageSrc={item.image}
-                title={item.title}
-                alt={item.alt || ""}
-                size={props.size}
-              />
-            </li>
-          );
-        }
-      })}
-    </ul>
+    <>
+      {/* mobile-view */}
+      <ul className={classNames("flex md:hidden", props.className)}>
+        {props.items.slice(0, 2).map((item, idx) => {
+          if (item.image != null) {
+            return (
+              <li key={idx} className="-mr-2 inline-block">
+                <Avatar
+                  className={props.border}
+                  imageSrc={item.image}
+                  title={item.title}
+                  alt={item.alt || ""}
+                  size={props.size}
+                />
+              </li>
+            );
+          }
+        })}
+        <li className="-mr-2 inline-block">
+          <button className="cursor-default">
+            <span className="flex h-9 w-9 items-center justify-center rounded-full border-2 border-white bg-gray-100 dark:border-gray-900">
+              <p className="text-gray-800">+{props.items.length - 2}</p>
```

Comment:
```suggestion
              {props.items.length-2 >0 && <p className="text-gray-800">+{props.items.length - 2}</p>}
```

Leaving this as a suggestion as I don't have a laptop right now

Reply:
Looking at this again we should be able to just remove the whole list element